modules/react/select/lib/hooks/useSelectInput.ts

```diff
@@ -62,6 +62,31 @@ export const useSelectInput = composeHooks(
         // eslint-disable-next-line react-hooks/exhaustive-deps
       }, [model.state.inputRef, model.state.items.length]);
 
+      React.useEffect(() => {
```

Comment:
this code is copied from `useComboboxInput` but I need to pass `textInputRef` here instead because `inputRef` on the model refers to the hidden input which is why I *think* it wasn't working.